content/blog/2021/08/19/WritingFastConstraintStreamsSecretRecipe.adoc

```diff
@@ -0,0 +1,119 @@
+= Constraint Streams get some more love
```

Comment:
How about "Constraint Streams: Faster, Harder, Stronger"? This title is good too though :)